app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -415,3 +423,17 @@ fun ToggleIconButton(
     }
 }
 
+@Composable
+fun TurnFinishedNotification() {
```

Comment:
Indeed it wasn't the primary intent of my pr, however I honestly couldn't stand not knowing whether the current turn had ended after the bid and was waiting for the next one or not, so I included it to.
For your second point indeed I though so as well so the code shouldn't be too hard to adapt to generate a more global summary. Moreover, I chose an AlertDialog because they have the nice property to close automatically if one clicks outside of them!